src/ripple/app/ledger/InboundLedgers.h

```diff
@@ -63,7 +63,7 @@ class InboundLedgers
 
     virtual int getFetchCount (int& timeoutCount) = 0;
 
-    virtual void logFailure (uint256 const& h) = 0;
+    virtual void logFailure (uint256 const& h, std::uint32_t seq) = 0;
```

Comment:
Is this a ledger sequence number?  spell it out a bit, perhaps.

We used to have a typedef for that - what happened?